lib/install/vchlog/vchlogger.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vchlog
+
+import (
+	"context"
+	"path"
+
+	"github.com/vmware/govmomi/object"
+	"github.com/vmware/vic/pkg/trace"
+)
+
+// VCHCreatedSignal serves as a signal struct indicating datastore folder path is available
```

Comment:
Can we come up with a more accurate name for this? (We're sending the datastore is created, not the VCH; we're only part of the way through the VCH creation process.)

Reply:
thanks, good catch